api/src/main/java/org/openmrs/util/OpenmrsUtil.java

```diff
@@ -1209,7 +1209,6 @@ public static void saveDocument(Document doc, File outFile) {
 		for (String token : tokens) {
 			token = token.trim();
 			if (token.length() == 0) {
-				continue;
```

Comment:
@dilekamadushan rewrite the if/else block to simply

```java
if (!token.isEmpty()) {
  ret.add(Integer.valueOf(token));
}
```

Reply:
@teleivo ok I will modify it. Thanks